### PR TITLE
Danny elena

### PR DESCRIPTION
**Pull request by Danny20050123**
# results:
Confusion Matrix:
[[3905  257]
[1108 4319]]
Accuracy: 0.8576
F1 Score: 0.8635

### CODE REVIEW

**Review by margoglvz — CHANGES_REQUESTED** [review]
Hi guys! Good work so far on the model. Although I do have some questions and here is some feedback:

- Make code more clear into sections:
  - [ ] Preprocessing (another file)
  - [ ] Training (another file)
  - [ ] Testing (another file)
  - [ ] OR combination of any but make clear names
  - This will make things a lot clearer to read and easier to fix in the long run. You can put all of these into one file if you wish but please make it into more clear **separate** code boxes
  - preprocessing → train/val split → training → final separate test evaluation
- [ ] Preprocessing
  - When you are using the testing dataset, please make sure to run those images **through your preprocessing code** (filters, resizing, etc). Since you did that for your training dataset you also have to do that with your testing dataset in order to actually calculate your model's performance and metrics. 
  - If you wish to simply to use different filters than your previous preprocessing code that is fine but please lmk what changed
 - **QUESTION** "filtered images" folder
   - Does this folder contain the filtered/preprocessed images of the training dataset **or** the testing dataset? 
   - Make sure that you have 2 separate folders that has your preprocessed training images and your preprocessed testing images
- [x] Make sure all imports are at the top of each code block

   
   
 **Summary**:
 Please make sure that for your model testing you are using the **ISIC 2019 testing dataset** because right now from what I am seeing you are using part of the training dataset as your training and validation set **as well** as your final testing dataset.  **This means that the model is testing with exact images it has already seen.** Overall CNN structure and testing code is good but make sure the whole flow is more organized and to use the correct datasets. If I am misunderstanding something tho please lmk! Thank you!
